mm2src/coins/utxo/utxo_builder/utxo_coin_builder.rs

```diff
@@ -571,7 +571,13 @@ pub trait UtxoCoinBuilderCommonOps {
     }
 
     #[inline]
-    fn check_utxo_maturity(&self) -> bool { self.activation_params().check_utxo_maturity.unwrap_or_default() }
+    fn check_utxo_maturity(&self) -> bool {
```

Comment:
Should it still be `inline`?

Reply:
Thank you! Referring to this comment https://github.com/KomodoPlatform/atomicDEX-API/pull/1302#discussion_r894296192, there is no need to mark these trait methods inline, since they're used in `coins` crate only. Even if `utxo-coin-builder` is moved to a separate crate.
I removed all of them.